packages/clay-badge/src/ClayBadge.soy

```diff
@@ -0,0 +1,69 @@
+{namespace ClayBadge}
+
+/**
+ * This renders a badge.
+ */
+{template .render}
+	{@param? closeable: bool}
+	{@param? href: string}
+	{@param? id: string}
+	{@param? label: html|string}
+	{@param? size: string}
+	{@param? spritemap: string}
+	{@param? style: string}
+
+	{let $classes kind="text"}
+		badge badge-pill
+		{sp}badge-{$style ?: 'default'}
+
+		{if $size}
+			{sp}badge-{$size}
+		{/if}
+	{/let}
+
+	{let $attributes kind="attributes"}
+		class="{$classes}"
+
+		{if $id}
+			id="{$id}"
+		{/if}
+	{/let}
+
+	{if $href}
+		{call ClayLink.render}
+			{param elementClasses: $classes /}
+			{param href: $href /}
+			{param id: $id /}
+			{param label: $label /}
+		{/call}
+	{else}
+		<span {$attributes}>
+			{call .content}
+				{param closeable: $closeable /}
+				{param label: $label /}
+				{param spritemap: $spritemap /}
+			{/call}
+		</span>
+	{/if}
+
+{/template}
+
+/**
+ * Renders icon content
+ */
+{template .content}
+	{@param? closeable: bool}
+	{@param? label: html|string}
+	{@param? spritemap: string}
+
+	{$label ? $label : ''}
+
+	{if $closeable and $spritemap}
+		<button aria-label="Close" class="close" type="button">
```

Comment:
How are we localizing hard-coded strings?

Reply:
We can locate by the metal selector, for example:
```javascript
events: {
	click: {
		selector: '.close',
		fn: handleClick
	}
}
```

Reply:
I meant `aria-label="Close"` will not be translated. Are there other components where we have hard-coded strings? If so, what's the pattern we are following?

Reply:
Talking with @jbalsas we think we should for now go with it hardcoded while we find the best solution. It's an interesting issue...

Reply:
Yeah :)

In portal, we have a mechanism in place that will translate keys used with
```soy
{msg desc=""}lang-key{/msg}```

Not sure we could use it here, though

Reply:
Yep, the point is that if we decide to go in that direction we should provide some "easy" mechanism to allow users to pass the keys to the component, or a series of .json with them that the build process can use... I don't know, just thinking loud... :P